stixpy/coordinates/frames.py

```diff
@@ -0,0 +1,161 @@
+import astropy
+import astropy.coordinates as coord
+import astropy.units as u
+from astropy.coordinates import QuantityAttribute
+from astropy.wcs import WCS
+from sunpy.coordinates.frameattributes import ObserverCoordinateAttribute
+from sunpy.coordinates.frames import HeliographicStonyhurst, SunPyBaseCoordinateFrame
+from sunpy.sun.constants import radius as _RSUN
+
+from stixpy.net.client import STIXClient  # noqa
+from stixpy.utils.logging import get_logger
+
+logger = get_logger(__name__, "DEBUG")
+
+__all__ = ["STIXImaging"]
+
+STIX_X_SHIFT = 26.1 * u.arcsec  # fall back to this when non sas solution available
+STIX_Y_SHIFT = 58.2 * u.arcsec  # fall back to this when non sas solution available
+STIX_X_OFFSET = 60.0 * u.arcsec  # remaining offset after SAS solution
+STIX_Y_OFFSET = 8.0 * u.arcsec  # remaining offset after SAS solution
+
```

Comment:
As an example, the below could be added here and then a find & replace of "SXLN-TAN" -> STIX_X_CTYPE could be done throughout PR.
```suggestion
STIX_X_CTYPE = "SXLN-TAN"
STIX_Y_CTYPE = "SXLT-TAN"

```

Just a suggestion if we think this will be useful.  If not, no problem.

Reply:
Done